functions/Get-DbaDbEncryption.ps1

```diff
@@ -136,6 +136,7 @@ function Get-DbaDbEncryption {
                         $returnCertificate.Owner = $serverCertificate.Owner
                         $returnCertificate.Object = $serverCertificate
                         $returnCertificate.ExpirationDate = $serverCertificate.ExpirationDate
+                        $returnCertificate.EncryptionAlgorithm = $db.DatabaseEncryptionKey.Properties.where( { $psitem.name -eq 'EncryptionAlgorithm' }).value
```

Comment:
At this time we are still supporting PowerShell v3, so this will need to be adjusted to use `Where-Object` instead of the where method.

Reply:
Edited to use where-object